src/index.js

```diff
@@ -1,8 +1,7 @@
 import Animate from './Animate';
 import { configBezier, configSpring } from './easing';
-import { translateStyle } from './util';
 import AnimateGroup from './AnimateGroup';
 
-export { configSpring, configBezier, AnimateGroup, translateStyle };
```

Comment:
Removing this export will break when recharts imports it: https://github.com/recharts/recharts/blob/master/src/component/Tooltip.tsx#L5

which is why I would do a major release.